agents/labjack/labjack_agent.py

```diff
@@ -167,20 +174,32 @@ def __init__(self, agent, ip_address, active_channels, function_file,
                                               function_file)
             with open(function_file_path, 'r') as stream:
                 self.functions = yaml.safe_load(stream)
+                if self.functions is None:
+                    self.functions = {}
                 print(f"Applying conversion functions: {self.functions}")
 
         self.initialized = False
         self.take_data = False
 
-        # Register feed
+        # Register main feed. Exclude influx due to potentially high scan rate
         agg_params = {
             'frame_length': 60,
+            'exclude_influx': True
         }
-        self.agent.register_feed('Sensors',
+        self.agent.register_feed('sensors',
                                  record=True,
                                  agg_params=agg_params,
                                  buffer_time=1)
 
+        # Register downsampled feed for influx.
+        agg_params_downsampled = {
+            'frame_length': 60
+        }
+        self.agent.register_feed('sensors_downsampled',
```

Comment:
Yeah we don't use the feed tag, so my vote is that this isn't a problem. And I think not many other institutions have started using the labjack yet, so I think we are the main people this will affect.